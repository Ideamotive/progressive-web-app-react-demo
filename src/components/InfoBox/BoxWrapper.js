import React from 'react';
import styled from 'styled-components';

export default styled.div`
    position: relative;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    margin-top: 10px;
    display: flex;
    top: 150px;
    left: 0;
    width: 300px;
    border-radius: 5px;
    flex-direction: column;
    background-color: #fff; 
    box-shadow: 0px 1px 4px 0px rgba(0,0,0,0.75);
    z-index: 10;
    `;
