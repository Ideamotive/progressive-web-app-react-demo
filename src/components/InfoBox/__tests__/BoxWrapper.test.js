import React from 'react';
import * as renderer from 'react-test-renderer';

import BoxWrapper from '../BoxWrapper';

describe('<BoxWrapper/> component', () => {
  test('should match to snapshot', () => {
    const tree = renderer
      .create(
        <BoxWrapper>
          {'test'}
        </BoxWrapper>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

