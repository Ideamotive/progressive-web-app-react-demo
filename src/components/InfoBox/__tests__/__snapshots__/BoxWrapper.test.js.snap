// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BoxWrapper/> component should match to snapshot 1`] = `
<div
  className="sc-bdVaJa dxyeMJ"
>
  test
</div>
`;
